hibernate-core/src/main/java/org/hibernate/sql/InsertSelect.java

```diff
@@ -20,7 +20,7 @@
 public class InsertSelect {
 	private String tableName;
 	private String comment;
-	private List columnNames = new ArrayList();
+	private ArrayList<String> columnNames = new ArrayList<>();
```

Comment:
Do we need ArrayList field type specification? In previous JDK version there is minor performance implication but in recent JDK the difference is eliminated, unless I were wrong.

Reply:
You prefer List<String> instead of ArrayList<String>? Yes, I think it is better too. We should change it.